### ISSUE TITLE
mAP on COCO

### ISSUE THREAD

**Issue by Crazod** [ueanperfect/SparseInst_mmdet_dev3.x#1]
Hi, Thanks for your repo. 
I want to test this repo for my project.
Have you tested on coco. What's map in this repo?

**Comment by ueanperfect** [comment]
> Hi, Thanks for your repo. I want to test this repo for my project. Have you tested on coco. What's map in this repo?

I'm afraid this repo does not support testing for the time being. This repo is only a backup of my code for developing this network. It only supports training and rest. If you have an emergency, you can temporarily test your dataset. However, I haven't tested it on the coco dataset, and.If you are not in a hurry, you can wait for the following. It is expected that it will be completed by the end of September with full functions and merged to the mmlab/mmdetection. At that time, you can directly install mmdetection_ Dev3. X framework runs SparseInst.

**Comment by Crazod** [comment]
> > Hi, Thanks for your repo. I want to test this repo for my project. Have you tested on coco. What's map in this repo?
> 
> I'm afraid this repo does not support testing for the time being. This repo is only a backup of my code for developing this network. It only supports training and rest. If you have an emergency, you can temporarily test your dataset. However, I haven't tested it on the coco dataset, and.If you are not in a hurry, you can wait for the following. It is expected that it will be completed by the end of September with full functions and merged to the mmlab/mmdetection. At that time, you can directly install mmdetection_ Dev3. X framework runs SparseInst.

Thanks. Will you pull MR to mmdetection ? Are you mmlab members? I used to work in sensetime. It may be need some time to merge in.

**Comment by ueanperfect** [comment]
> > > Hi, Thanks for your repo. I want to test this repo for my project. Have you tested on coco. What's map in this repo?
> > 
> > 
> > I'm afraid this repo does not support testing for the time being. This repo is only a backup of my code for developing this network. It only supports training and rest. If you have an emergency, you can temporarily test your dataset. However, I haven't tested it on the coco dataset, and.If you are not in a hurry, you can wait for the following. It is expected that it will be completed by the end of September with full functions and merged to the mmlab/mmdetection. At that time, you can directly install mmdetection_ Dev3. X framework runs SparseInst.
> 
> Thanks. Will you pull MR to mmdetection ? Are you mmlab members? I used to work in sensetime. It may be need some time to merge in.

Yes I am a mmlab member now.And this network will be merged in mmdetection in the future.

**Comment by Crazod** [comment]
Thanks a lot

**Comment by Crazod** [comment]
> > > > Hi, Thanks for your repo. I want to test this repo for my project. Have you tested on coco. What's map in this repo?
> > > 
> > > 
> > > I'm afraid this repo does not support testing for the time being. This repo is only a backup of my code for developing this network. It only supports training and rest. If you have an emergency, you can temporarily test your dataset. However, I haven't tested it on the coco dataset, and.If you are not in a hurry, you can wait for the following. It is expected that it will be completed by the end of September with full functions and merged to the mmlab/mmdetection. At that time, you can directly install mmdetection_ Dev3. X framework runs SparseInst.
> > 
> > 
> > Thanks. Will you pull MR to mmdetection ? Are you mmlab members? I used to work in sensetime. It may be need some time to merge in.
> 
> Yes I am a mmlab member now.And this network will be merged in mmdetection in the future.

Hi， How's it going now. I try to use this repo but can't obtain a worked ap.

**Comment by ueanperfect** [comment]
really sorry to reply so late, I am currently bussy with my courseworks in school, maybe It seems this repo have some issues,  especially with the data preprocessing. maybe I can fix them in the future.
